### PR TITLE
Add removed parts, improve structure, add metainfo

### LINKED ISSUE #13: `Structure of the Document` needs adjustment
**Describe the bug**
Currently the `Structure of the Document` in [IDTA-01005_Introduction.adoc](https://github.com/admin-shell-io/aas-specs-aasx/blob/IDTA-01005-3-1_working/documentation/IDTA-01005/modules/ROOT/pages/IDTA-01005_Introduction.adoc ) still refers on numbered clauses, which is not the case anymore. Also the structure has been adjusted in c0ae8e26cab4bf3188a0a4f10a905dfbf9705880: `Introduction.adoc` includes now `Preamble`, `Introduction` and `Basic Concepts and Leading Picture`.

How should we proceed? Should we split `Introduction.adoc` to three docs and thus also clauses or should we adapt the text?

**Where**
https://github.com/admin-shell-io/aas-specs-aasx/blob/IDTA-01005-3-1_working/documentation/IDTA-01005/modules/ROOT/pages/IDTA-01005_Introduction.adoc)

### LINKED ISSUE #14: `Terms and definitions` are missing in the documentation
**Describe the bug**
In c0ae8e26cab4bf3188a0a4f10a905dfbf9705880: the  `terms and definitions` part was removed. Was it on purpose?

If yes, we should adjust the text in the `Structure of the Document` in [IDTA-01005_Introduction.adoc](https://github.com/admin-shell-io/aas-specs-aasx/blob/IDTA-01005-3-1_working/documentation/IDTA-01005/modules/ROOT/pages/IDTA-01005_Introduction.adoc )

**Where**
https://github.com/admin-shell-io/aas-specs-aasx/blob/IDTA-01005-3-1_working/documentation/IDTA-01005/modules/ROOT/pages/IDTA-01005_Introduction.adoc)

### PR DESCRIPTION
**Pull request by zrgt**
- Add removed parts when the document was divided in c0ae8e26cab4bf3188a0a4f10a905dfbf9705880
- Split `introduction.adoc` in `introduction.adoc` and `basic-concepts-and-leading-picture.adoc`
- Add metainfo in `index.adoc` similar to part 1 documentation

fix #13, fix #14;

### CODE REVIEW

File: documentation/IDTA-01005/modules/ROOT/nav.adoc [hunk 1]
```diff
@@ -10,10 +10,13 @@ Illustrations:
...
 
+* xref:terms-definitions-and-abbreviations.adoc[Terms and Definitions]
+* xref:preamble.adoc[Preamble]
 * xref:introduction.adoc[Introduction]
-* xref:aasx.adoc[Specification]
+* xref:basic-concepts-and-leading-picture.adoc[Basic Concepts and Leading Picture]
+* xref:aasx.adoc[AASX Specification]
 * xref:summary-and-outlook.adoc[Summary and Outlook]
-* xref:./annex/nav_annex.adoc[Annex]
+* Annex
```
Comment (BirgitBoss) [inline documentation/IDTA-01005/modules/ROOT/nav.adoc:19]: why did you remove nav_annex.adoc? It looks the same, yes, but it has advantages to have this nav-file

**Review by BirgitBoss — APPROVED** [review]

File: documentation/IDTA-01005/modules/ROOT/nav.adoc (hunk 1 above)
Comment (zrgt) [reply documentation/IDTA-01005/modules/ROOT/nav.adoc:19]: The file is empty. Could you explain the advatages? 

If any text should appear in the top page of Annex, we can always add the .adoc file for that.
